tools/sgen/output/FluentUIStyle.generated.swift

```diff
@@ -0,0 +1,2958 @@
+/// Autogenerated file
```

Comment:
Does the output need to be in git?  This would make a duplicate copy if we consume it from library?

Reply:
I am mostly concerned about manual edits to this file making it out of sync with original yml file

Reply:
We don't need the output to be there, but I think it'll make it easier for us to spot changes on the output as we make changes in the YAML file.
But I agree we shouldn't manually edit this file.